inbox/sqlalchemy_ext/util.py

```diff
@@ -6,12 +6,6 @@
 import weakref
 from typing import Any, MutableMapping, Optional, Tuple
 
-from bson import EPOCH_NAIVE, json_util
-
-# Monkeypatch to not include tz_info in decoded JSON.
-# Kind of a ridiculous solution, but works.
-json_util.EPOCH_AWARE = EPOCH_NAIVE
```

Comment:
Since this was indeed ridiculous I just inlined the monkeypatched code into the vendored module.